### DISCUSSION TITLE
Autodocs should also check for README file

### DISCUSSION THREAD

**Discussion by cssinate** [storybookjs/storybook#28885] [Ideas]
### Is your feature request related to a problem? Please describe.

In our design system, each component is its own package with its own README file. We add the same text to the README as we add to the story for the main file. Updating the description means updating in both files.

### Describe the solution you'd like

If there is no description generated from the component file, and there is a README in the same directory, the README should be used as the auto-generated description for the story.

### Describe alternatives you've considered

We've considered a run-time script that copies text from one place to another, but thought that it would be a useful feature for other users. [There's this integration](https://storybook.js.org/addons/storybook-readme) that was very popular, but is no longer maintained and doesn't work on later versions of Storybook or React. But the popularity of it shows that it would be a useful feature for other users.

### Are you able to assist to bring the feature to reality?

no

### Additional context

My current answer for being able to help is a "no", but I may try to prioritize it if it doesn't get enough interest.

**Comment by smlimasu** [comment]
Could you not just make an autodocs template that conditionally renders the markdown readme as markdown docs block if no description is present?

**Comment by cssinate** [comment]
Good idea. That might work. Thanks.
